photon-ml/src/main/scala/com/linkedin/photon/ml/diagnostics/hl/PredictedProbabilityVersusObservedFrequencyHistogramBin.scala

```diff
@@ -44,19 +42,19 @@ class PredictedProbabilityVersusObservedFrequencyHistogramBin(
     var observedNegCount: Long = 0L)
   extends Serializable {
 
-  Preconditions.checkArgument(lowerBound >= 0 && lowerBound <= 1.0)
-  Preconditions.checkArgument(upperBound >= 0 && upperBound <= 1.0)
-  Preconditions.checkArgument(lowerBound < upperBound)
-  Preconditions.checkArgument(observedNegCount >= 0L)
-  Preconditions.checkArgument(observedPosCount >= 0L)
+  require(lowerBound >= 0 && lowerBound <= 1.0)
+  require(upperBound >= 0 && upperBound <= 1.0)
+  require(lowerBound < upperBound)
+  require(observedNegCount >= 0L)
+  require(observedPosCount >= 0L)
```

Comment:
I recall that earlier you described the difference between assert and require as (paraphrasing) "require() when arguments are provided by the user, assert() when they're provided from elsewhere in the code". With that in mind, shouldn't these be asserts?